<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="9" passed="7" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-18T16:38:12 IST" name="Default suite" finished-at="2024-03-18T16:40:22 IST" duration-ms="130563">
    <groups>
    </groups>
    <test started-at="2024-03-18T16:38:12 IST" name="Default test" finished-at="2024-03-18T16:40:22 IST" duration-ms="130563">
      <class name="Sanity.Cell_annotation.Testing">
        <test-method is-config="true" signature="setup()[pri:0, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:38:12 IST" name="setup" finished-at="2024-03-18T16:38:14 IST" duration-ms="2003" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="login()[pri:1, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:38:14 IST" name="login" finished-at="2024-03-18T16:38:32 IST" duration-ms="17755" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="series_set()[pri:2, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:38:32 IST" name="series_set" finished-at="2024-03-18T16:38:54 IST" duration-ms="22548" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- series_set -->
        <test-method signature="cellannotation()[pri:3, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:38:54 IST" name="cellannotation" finished-at="2024-03-18T16:38:59 IST" duration-ms="4584" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cellannotation -->
        <test-method signature="Tileloading()[pri:4, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:38:59 IST" name="Tileloading" finished-at="2024-03-18T16:39:09 IST" duration-ms="10163" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Tileloading -->
        <test-method signature="centroidDetection()[pri:5, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:39:09 IST" name="centroidDetection" finished-at="2024-03-18T16:39:22 IST" duration-ms="13260" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Relearn failed] but found [Relearn success]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Relearn failed] but found [Relearn success]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Sanity.Cell_annotation.Testing.centroidDetection(Testing.java:261)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- centroidDetection -->
        <test-method signature="celldetection()[pri:6, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:39:22 IST" name="celldetection" finished-at="2024-03-18T16:39:42 IST" duration-ms="19846" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- celldetection -->
        <test-method signature="Rabies()[pri:7, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:39:42 IST" name="Rabies" finished-at="2024-03-18T16:39:53 IST" duration-ms="11159" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Relearn failed] but found [Relearn success]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Relearn failed] but found [Relearn success]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Sanity.Cell_annotation.Testing.Rabies(Testing.java:299)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Rabies -->
        <test-method signature="Annotation()[pri:8, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:39:53 IST" name="Annotation" finished-at="2024-03-18T16:40:06 IST" duration-ms="12971" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Annotation -->
        <test-method signature="save()[pri:9, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:40:06 IST" name="save" finished-at="2024-03-18T16:40:22 IST" duration-ms="16015" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- save -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:Sanity.Cell_annotation.Testing@290222c1]" started-at="2024-03-18T16:40:22 IST" name="afterTest" finished-at="2024-03-18T16:40:22 IST" duration-ms="197" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- Sanity.Cell_annotation.Testing -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
